js/scripts.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function() {
+	function Button() {
+		this.text = text || "Hello";
+	}
+	Button.prototype = {
+		create: function() {
+			var self = this;
+			this.$element = $("<button>");
+			this.$element = $(this.text);
```

Comment:
you cannot select by jQuery this.text because this is just a text - you should assign `this.text` to `this.$element` by method `text();`